src/renderer/Components/Introduction/index.jsx

```diff
@@ -0,0 +1,56 @@
+import deliTruck from "@images/deliTruck.png";
+
+function Introduction() {
+  return (
+    <div className="flex flex-1 items-center justify-center overflow-y-auto bg-white p-6">
+      <div className="flex w-full max-w-7xl overflow-hidden rounded-3xl shadow-2xl">
+        <div className="to-blue-medium flex flex-1 flex-col justify-between bg-gradient-to-br from-sky-500 p-12 text-white">
+          <div>
+            <h1 className="mb-6 text-5xl font-extrabold leading-tight">
+              파일 작업 <span className="text-yellow-bright">자동화</span>{" "}
+              패키지
+            </h1>
+            <p className="mb-6 text-2xl font-light">
+              원격 조종이나 복잡한 메뉴얼은 필요 없습니다.
+            </p>
+            <p className="mb-8 text-xl font-light">
+              파일 작업 자동화 패키지를 만들고,
+              <br /> 모든 사람과 쉽게 공유하고 실행하세요!
+            </p>
+          </div>
+          <img
+            src={deliTruck}
+            className="mx-auto mb-8 mt-8 max-w-md"
+            alt="Product Image"
+          />
+          <div className="flex justify-center space-x-4"></div>
+        </div>
+        <div className="flex flex-1 flex-col justify-center bg-gray-50 p-12">
+          <h2 className="mb-8 text-4xl font-bold text-gray-800">주요 기능</h2>
+          <ul className="space-y-6">
+            {[
+              { icon: "📂", text: "파일 전송이 가능합니다" },
+              { icon: "📂", text: "파일 이동이 가능합니다" },
+              { icon: "📂", text: "파일 복제가 가능합니다" },
+              { icon: "🗑️", text: "파일 삭제가 가능합니다" },
+              { icon: "🚀", text: "파일 실행이 가능합니다" },
+              { icon: "📦", text: "압축 해제가 가능합니다" },
+            ].map((item, index) => (
```

Comment:
해당 기능 리스트를 일반 변수로 빼는것은 어떻게 생각하시나여 ? 😁
```js
const features = [
              { icon: "📂", text: "파일 전송이 가능합니다" },
              { icon: "📂", text: "파일 이동이 가능합니다" },
              { icon: "📂", text: "파일 복제가 가능합니다" },
              { icon: "🗑️", text: "파일 삭제가 가능합니다" },
              { icon: "🚀", text: "파일 실행이 가능합니다" },
              { icon: "📦", text: "압축 해제가 가능합니다" },
            ]
```

```suggestion
            {features.map((item, index) => (
```

Reply:
변수로 따로 빼서 관리하면 추가될때에도 관리하기가 쉬울듯합니다. 개선해보겠습니다!